app/helpers/certificate_helper.rb

```diff
@@ -0,0 +1,13 @@
+module CertificateHelper
+  def linkedin_url_to_post(certificate)
+    URI::HTTPS.build host: 'www.linkedin.com',
+                     path: '/profile/add',
+                     query: Rack::Utils.build_query(startTask: 'CERTIFICATION_NAME',
+                                                    name: certificate.title,
+                                                    organizationId: 3648837,
```

Comment:
We should extract this to a conf or use an env variable

Reply:
MUMUKI_LINKEDIN_ORGANIZATION_ID